repos/system_upgrade/common/libraries/rhui.py

```diff
@@ -112,6 +112,17 @@
                 ('leapp-google-sap.repo', YUM_REPOS_PATH)
             ],
         },
+        'alibaba': {
+            'src_pkg': 'client-rhel7',
+            'target_pkg': 'aliyun_rhui_rhel8.6',
```

Comment:
Could you please elaborate on this `aliyun_rhui_rhel8.6` package? For a correct functionality we require not versioned RHUI client package which contains repositories using $releasever. Otherwise we would have to always change target_pkg for very release and could not support multiple upgrade paths.

Reply:
This package name contains version information, which may lead to ambiguity as it only applies to rhel 8.6. In fact, it applies to all rhel 8 versions, and the source address used $releasever,so please ignore the target pkg name